### DISCUSSION TITLE
How to remove all console calls?

### DISCUSSION THREAD

**Discussion by Lyb-shoujibuyaofangkoudai** [nuxt/nuxt#28386] [Questions]
I've tried almost every method available online, including third-party plugins, Vite plugins, Vite configurations, and even wrote a Nuxt plugin to forcibly replace the console methods in window, but none of these approaches worked. I'm not sure if it's due to my Node.js version or the Nuxt version I'm using. Therefore, I hope to have a feature that removes all console calls.
node: 20.10.0
nuxt: 3.10.3

**Comment by xjccc** [comment]
You could try this vite plugin.

[vite-plugin-remove-console](https://www.npmjs.com/package/vite-plugin-remove-console)
